src/apiCalls/transactions/sendSignedBatchTransactions.ts

```diff
@@ -0,0 +1,60 @@
+import axios from 'axios';
+import { TIMEOUT } from 'constants/network';
+import { buildBatchId } from 'hooks/transactions/helpers/buildBatchId';
+import { addressSelector, networkSelector } from 'reduxStore/selectors';
+import { store } from 'reduxStore/store';
+import {
+  BatchTransactionsRequestType,
+  BatchTransactionsResponseType,
+  CustomTransactionInformation,
+  SignedTransactionType
+} from 'types';
+import { TRANSACTIONS_BATCH } from '../endpoints';
+
+export interface SendBatchTransactionsPropsType {
+  transactions: SignedTransactionType[][];
+  sessionId: string;
+  customTransactionInformationOverrides?: Partial<CustomTransactionInformation>;
+}
+
+export type SendSignedBatchTransactionsReturnType = {
+  error?: string | null;
+  batchId?: string | null;
+  data?: BatchTransactionsResponseType;
+};
+
+export async function sendSignedBatchTransactions({
+  transactions,
+  sessionId
+}: SendBatchTransactionsPropsType) {
+  const address = addressSelector(store.getState());
+  const { apiAddress, apiTimeout } = networkSelector(store.getState());
```

Comment:
if (!address) {
return;
}

Reply:
Not necessary. This function will be called only in a logged-in state